src/main/java/uk/gov/hmcts/ccd/data/SignificantItemEntity.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.ccd.data;
+
+import uk.gov.hmcts.ccd.data.casedetails.CaseAuditEventEntity;
+import uk.gov.hmcts.ccd.domain.model.callbacks.ItemType;
+
+import javax.persistence.*;
+import java.net.URL;
+@Table(name = "case_event_significant_items")
```

Comment:
Formatting: Empty line between imports and class decorators.

Reply:
Addressed

Reply:
addressed

Reply:
I have removed it but seems to not reflect here